doc/source/io.rst

```diff
@@ -4777,7 +4777,7 @@ writes ``data`` to the database in batches of 1000 rows at a time:
     data.to_sql('data_chunked', engine, chunksize=1000)
 
 SQL data types
-++++++++++++++
```

Comment:
do the docs look ok after this?

Reply:
This is moving it one level up (since this is the only 'subsubsection' in the while sql section, I think that is fine). @mroeschke what was your reasoning here?

Reply:
Ah. I thought this was rendering incorrectly when I built the docs, but I was looking at the wrong section. I'll revert it.